### DISCUSSION TITLE
run V while in linux with virtualbox vm of window 10

### DISCUSSION THREAD

**Discussion by gshields** [vlang/v#11881] [Ideas]
Running V as a mounted drive to  a window partition which is thne mounted into a virtual box (windows 10), 
V was compiled with gcc 10 (mingw),
this previously work with a v version from months ago, and now I get 

Z:\v>v.exe
Welcome to the V REPL (for help with V itself, type `exit`, then run `v help`).
failed    cmd: "UNC\VBoxSvr\Portableapps\v\v.exe" "Z:\v\UNC\VBoxSvr\Portableapps\v\cmd\tools\vrepl.v"
failed   code: 2
V panic: exec failed (CreateProcess) with code 2: The system cannot find the file specified.
 cmd: "UNC\VBoxSvr\Portableapps\v\v.exe" "Z:\v\UNC\VBoxSvr\Portableapps\v\cmd\tools\vrepl.v"
v hash: aaee251
print_backtrace_skipping_top_frames is not implemented

**Comment by JalonSolov** [comment]
Did you run `v symlink` in the VM?  If you didn't it won't have the path to V, so you'll get that type of message.

Also, make sure you're on the actual latest version of V, not V from a .zip file.  Those are out of date almost immediately.
